source/blog/2007-10-09-first-website.md

```diff
@@ -3,7 +3,7 @@ title: "New Website"
 authorName: jwage
 authorEmail:
 categories: []
-permalink: /2007/10/09/new-website.html
+permalink: /2007/10/09/first-website.html
```

Comment:
I had to rename the permalink because "new-website" is a slug that exists two times for blogs and won't work as ids otherwise without rewriting too much. Because this is an older (the first) blog post, it shouldn't have much of an impact on search engines.